ocaml/xapi/dune

```diff
@@ -278,3 +278,13 @@
   )
 )
 
+(rule
+  (alias runtest)
+  (package xapi)
+  (deps
+    (:x xapi_main.exe)
+    ../xenopsd/test/check-no-xenctrl.sh
+  )
+  (action (run ../xenopsd/test/check-no-xenctrl.sh %{x}))
```

Comment:
This test is already present in [ocaml/tests/dune](https://github.com/xapi-project/xen-api/blob/master/ocaml/tests/dune#L167), no need to add another one #5222 
It's good to know that xenops/cmxs doesn't need t obe tested anymore, though

Reply:
Ah I was looking under `ocaml/xapi` and couldn't find anything there. I removed the extra commit from here.